src/ImpactX.H

```diff
@@ -7,8 +7,9 @@
 #ifndef IMPACT_X_H
 #define IMPACT_X_H
 
-#include <AMReX_AmrCore.H>
+#include "particles/ImpactXParticleContainer.H"
 
+#include <AMReX_AmrCore.H>
```

Comment:
For `std::unique_ptr`:
```suggestion
#include <AMReX_AmrCore.H>

#include <memory>
```